.github/actions/release-branch/action.yaml

```
@@ -0,0 +1,86 @@
+name: "Create Release Branch from Tag"
```

```suggestion
name: "Release Branch"
```

We might allow this to update release branches in response to force-retags and other things